tests/src/test/java/org/eclipse/hono/tests/LegacyDeviceRegistryHttpClient.java

```diff
@@ -594,19 +593,20 @@ public LegacyDeviceRegistryHttpClient(final Vertx vertx, final String host, fina
     /**
      * Creates a tenant and adds a device to it with a given password.
      * <p>
-     * This method simply invokes {@link #addDeviceForTenant(TenantObject, String, JsonObject, String)}
+     * This method simply invokes {@link #addDeviceForTenant(String, JsonObject, String, JsonObject, String)}
```

Comment:
why do you want to replace the TenantObject here with a JsonObject? Is there a reason not to stick with the original code (which we will remove soon anyway)?